packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -64,6 +70,20 @@ class ToggleSectionHeader extends React.Component {
     this.wrapOnClick = this.wrapOnClick.bind(this);
   }
 
+  componentDidUpdate(previousProps, previousState) {
+    if (previousProps.isOpen !== this.props.isOpen) {
+      const { onOpen, onClose } = this.props;
+
+      if (onOpen && !previousState.isOpen) {
+        onOpen();
+      } else if (onClose && previousState.isOpen) {
+        onClose();
+      }
```

Comment:
if they are programmatically setting isOpen, do we need to trigger the onOpen or onClose callbacks?

Reply:
Removed dffedb0